packages/ui/src/components/form-input/index.ts

```diff
@@ -18,9 +16,9 @@ const FormInput = {
     onCheckedChange: field.onChange
   })),
   Select: withForm(Select) as <T = string>(
-    props: FormSelectProps<T> & { ref?: React.ForwardedRef<HTMLButtonElement> }
-  ) => React.ReactElement,
-  MultiSelect: FormMultiSelect
+    props: FormSelectProps<T> & { ref?: ForwardedRef<HTMLButtonElement> }
```

Comment:
This needs a clean up too. We can pick it later. Its only the typecasting which we need to remove.